.gitignore

```diff
@@ -1,4 +1,5 @@
-/attachments/
+/attachments/*
+!/attachments/.gitkeep
```

Comment:
Actually, would placing a .gitignore file that only contained '*' achieve the same effect here?